curriculum/challenges/english/01-responsive-web-design/css-box-model/part-007.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 60a3e3396c7b40068ad69970
+title: Part 7
+challengeType: 0
+dashedName: part-7
+---
+
+# --description--
+
+Before you can start styling the `div` you added, you need to link your CSS to your HTML.
+
+Add a `link` element to link your `styles.css` file, and remember to set the `rel` attribute to `stylesheet`.
```

Comment:
As `href` attribute is expected as `./styles.css` it might be good to have that pointed in the challenge description, not only in one test.